src/app/shell/personal-cabinet/provider/create-workshop/create-about-form/create-about-form.component.html

```diff
@@ -115,7 +115,7 @@
       Безкоштовно</mat-radio-button>
     <div fxLayout='row' fxLayoutAlign="start center" fxLayout.xs='column' fxLayoutAlign.xs="center start">
       <div fxLayout="row" fxLayoutAlign="center center">
-        <mat-radio-button #radio2 name="radioBtn2" [value]="true"><input matInput class="step-input step-input-age"
+        <mat-radio-button #radio2 name="radioBtn2" [value]="true"><input matInput class="step-input step-input-age price-input"
```

Comment:
remove syep-inout-age (this class is not needed anymore) and leave only step-price-input